oneanddone/tests/functional/pages/user/user_profile_edit.py

```diff
@@ -22,45 +20,49 @@ class UserProfileEditPage(Base):
 
     @property
     def bugzilla_email(self):
-        return self.selenium.find_element(
-            *self._bugzilla_email_input_locator).get_attribute('value')
+        return self.find_element(
+            self._bugzilla_email_input_locator).get_attribute('value')
+
+    @bugzilla_email.setter
```

Comment:
After PyPOM, maybe this is the next improvement to make across our suites. :smile: